app/src/service/ahgoraService.js

```diff
@@ -1,129 +1,123 @@
   // window.Neutralino.os.showMessageBox('Welcome', 'Hello Neutralinojs');
   // window.Neutralino.os.showNotification('Oops :/', 'Something went wrong', 'ERROR');
-   
 
   login: function (user) {
     return new Promise((resolve, reject) => {
       const credential = new URLSearchParams({
         empresa: user.company,
-        matricula: user.username, 
-        senha: user.password, 
-      });
+        matricula: user.username,
+        senha: user.password
+      })
 
-      let comand = `curl -d "${credential}" -X POST https://www.ahgora.com.br/externo/login`;
+      const comand = `curl -d "${credential}" -X POST https://www.ahgora.com.br/externo/login`
 
       window.Neutralino.os.execCommand(comand).then((result) => {
-        let userDetails = JSON.parse(result.stdOut);
-        if (userDetails.r === "success") {      
+        const userDetails = JSON.parse(result.stdOut)
+        if (userDetails.r === 'success') {
           StorageService.saveCredentials(user)
 
+          localStorage.setItem('userDetails', JSON.stringify(userDetails))
+          localStorage.setItem('credential', JSON.stringify(user))
 
-          localStorage.setItem("userDetails", JSON.stringify(userDetails));
-          localStorage.setItem("credential", JSON.stringify(user));
-
-          let comand = `curl https://www.ahgora.com.br/batidaonline/defaultComputer?c=${user.company}`;
+          const comand = `curl https://www.ahgora.com.br/batidaonline/defaultComputer?c=${user.company}`
           window.Neutralino.os.execCommand(comand).then((result) => {
-            localStorage.setItem("identity", result.stdOut);
+            localStorage.setItem('identity', result.stdOut)
           })
 
-
-          resolve(userDetails);
+          resolve(userDetails)
         } else {
-          reject(userDetails);
+          reject(userDetails)
         }
-      });
-    });
+      })
+    })
   },
   getProfileImg: function () {
-    let jwt = JSON.parse(localStorage.getItem("userDetails")).jwt;
-    let profileImg = JSON.parse(localStorage.getItem("userDetails")).employee_id;
-    const header = `cookie: qwert-external=${jwt}`;
-      
-    let comand = `curl -H "${header}" https://www.ahgora.com.br/externo/get_image/${profileImg} --output profile.jpg`;
-    console.log(comand)
-    window.Neutralino.os.execCommand(comand);
-
+    const jwt = JSON.parse(localStorage.getItem('userDetails')).jwt
+    const profileImg = JSON.parse(
+      localStorage.getItem('userDetails')
+    ).employee_id
+    const header = `cookie: qwert-external=${jwt}`
 
+    const comand = `curl -H "${header}" https://www.ahgora.com.br/externo/get_image/${profileImg} --output profile.jpg`
+    console.log(comand)
+    window.Neutralino.os.execCommand(comand)
   },
   espelhoPonto: function (year, month) {
-    let today = new Date();
-    let mirroDate = new Date(year,parseInt(month)-1,1);
+    const today = new Date()
+    const mirroDate = new Date(year, parseInt(month) - 1, 1)
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
    const mirroDate = new Date(year, parseInt(month) - 1, 1);
```